[2023-10-30T11:25:03.157+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: random_names.streaming_to_kafka scheduled__2023-10-30T11:20:00+00:00 [queued]>
[2023-10-30T11:25:03.170+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: random_names.streaming_to_kafka scheduled__2023-10-30T11:20:00+00:00 [queued]>
[2023-10-30T11:25:03.170+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2023-10-30T11:25:03.217+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): streaming_to_kafka> on 2023-10-30 11:20:00+00:00
[2023-10-30T11:25:03.222+0000] {standard_task_runner.py:57} INFO - Started process 352 to run task
[2023-10-30T11:25:03.225+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'random_names', 'streaming_to_kafka', 'scheduled__2023-10-30T11:20:00+00:00', '--job-id', '48', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpdcgz6t0f']
[2023-10-30T11:25:03.227+0000] {standard_task_runner.py:85} INFO - Job 48: Subtask streaming_to_kafka
[2023-10-30T11:25:03.273+0000] {task_command.py:415} INFO - Running <TaskInstance: random_names.streaming_to_kafka scheduled__2023-10-30T11:20:00+00:00 [running]> on host df75f57c0534
[2023-10-30T11:25:03.371+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='random_names' AIRFLOW_CTX_TASK_ID='streaming_to_kafka' AIRFLOW_CTX_EXECUTION_DATE='2023-10-30T11:20:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-30T11:20:00+00:00'
[2023-10-30T11:25:04.107+0000] {logging_mixin.py:151} INFO - {'full name': 'Mr Ananth Namnaik', 'gender': 'male', 'location': 'Sao Tome Old Quarter 3399', 'city': 'Morena', 'country': 'India', 'postcode': 90956, 'latitude': '12.0157', 'longtitude': '-163.6026', 'email': 'ananth.namnaik@example.com'}
[2023-10-30T11:25:04.115+0000] {conn.py:380} INFO - <BrokerConnection node_id=bootstrap-0 host=kafka:29092 <connecting> [IPv4 ('172.22.0.5', 29092)]>: connecting to kafka:29092 [('172.22.0.5', 29092) IPv4]
[2023-10-30T11:25:04.116+0000] {conn.py:1205} INFO - Probing node bootstrap-0 broker version
[2023-10-30T11:25:04.117+0000] {conn.py:410} INFO - <BrokerConnection node_id=bootstrap-0 host=kafka:29092 <connecting> [IPv4 ('172.22.0.5', 29092)]>: Connection complete.
[2023-10-30T11:25:04.224+0000] {conn.py:1267} INFO - Broker version identified as 2.5.0
[2023-10-30T11:25:04.224+0000] {conn.py:1268} INFO - Set configuration api_version=(2, 5, 0) to skip auto check_version requests on startup
[2023-10-30T11:25:04.231+0000] {logging_mixin.py:151} INFO - KafkaProducer connected successfully.
[2023-10-30T11:25:04.388+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:25:04.389+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:25:04.390+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:25:04.390+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:25:04.492+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:25:04.492+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:25:04.492+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:25:04.492+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:25:04.645+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:25:04.647+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:25:04.648+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:25:04.649+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:25:04.752+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:25:04.753+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:25:04.753+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:25:04.754+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:25:04.957+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:25:04.958+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:25:04.959+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:25:04.960+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:25:05.113+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:25:05.114+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:25:05.115+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:25:05.115+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:25:05.369+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:25:05.370+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:25:05.370+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:25:05.371+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:25:05.624+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:25:05.625+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:25:05.626+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:25:05.626+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:25:06.183+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:25:06.184+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:25:06.185+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:25:06.186+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:25:06.741+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:25:06.742+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:25:06.742+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:25:06.743+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:25:07.552+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:25:07.553+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:25:07.554+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:25:07.554+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:25:08.363+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:25:08.365+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:25:08.366+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:25:08.367+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:25:09.378+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:25:09.380+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:25:09.380+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:25:09.381+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:25:10.392+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:25:10.393+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:25:10.393+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:25:10.394+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:25:11.658+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:25:11.660+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:25:11.660+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:25:11.661+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:25:12.924+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:25:12.925+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:25:12.925+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:25:12.926+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:25:13.985+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:25:13.986+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:25:13.986+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:25:13.986+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:25:15.041+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:25:15.041+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:25:15.042+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:25:15.042+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:25:15.950+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:25:15.951+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:25:15.952+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:25:15.952+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:25:16.863+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:25:16.864+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:25:16.864+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:25:16.865+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:25:18.128+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:25:18.130+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:25:18.130+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:25:18.131+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:25:19.395+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:25:19.396+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:25:19.396+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:25:19.397+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:25:20.359+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:25:20.360+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:25:20.361+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:25:20.361+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:25:21.322+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:25:21.323+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:25:21.324+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:25:21.325+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:25:22.235+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:25:22.236+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:25:22.237+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:25:22.238+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:25:23.147+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:25:23.148+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:25:23.148+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:25:23.149+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:25:24.261+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:25:24.263+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:25:24.263+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:25:24.264+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:25:25.376+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:25:25.377+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:25:25.377+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:25:25.378+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:25:25.717+0000] {local_task_job_runner.py:115} ERROR - Received SIGTERM. Terminating subprocesses
[2023-10-30T11:25:25.726+0000] {process_utils.py:131} INFO - Sending 15 to group 352. PIDs of all processes in the group: [352]
[2023-10-30T11:25:25.727+0000] {process_utils.py:86} INFO - Sending the signal 15 to group 352
[2023-10-30T11:25:25.728+0000] {taskinstance.py:1630} ERROR - Received SIGTERM. Terminating subprocesses.
[2023-10-30T11:25:25.753+0000] {taskinstance.py:1935} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/scripts/script.py", line 52, in start_streaming
    future = producer.send('random_names', value=data)
  File "/home/airflow/.local/lib/python3.8/site-packages/kafka/producer/kafka.py", line 576, in send
    self._wait_on_metadata(topic, self.config['max_block_ms'] / 1000.0)
  File "/home/airflow/.local/lib/python3.8/site-packages/kafka/producer/kafka.py", line 699, in _wait_on_metadata
    metadata_event.wait(max_wait - elapsed)
  File "/usr/local/lib/python3.8/threading.py", line 558, in wait
    signaled = self._cond.wait(timeout)
  File "/usr/local/lib/python3.8/threading.py", line 306, in wait
    gotit = waiter.acquire(True, timeout)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1632, in signal_handler
    raise AirflowException("Task received SIGTERM signal")
airflow.exceptions.AirflowException: Task received SIGTERM signal
[2023-10-30T11:25:28.509+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:25:29.433+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:25:29.433+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:25:29.433+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:25:29.484+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:25:29.486+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:25:29.486+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:25:29.496+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:25:30.559+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:25:30.560+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:25:30.561+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:25:30.562+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:25:31.620+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:25:31.620+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:25:31.621+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:25:31.621+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:25:31.803+0000] {taskinstance.py:1398} INFO - Marking task as FAILED. dag_id=random_names, task_id=streaming_to_kafka, execution_date=20231030T112000, start_date=20231030T112503, end_date=20231030T112531
[2023-10-30T11:25:32.531+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:25:32.532+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:25:32.532+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:25:32.532+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:25:33.431+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:25:33.431+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:25:33.432+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:25:33.432+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:25:38.870+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:25:43.922+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:25:43.922+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:25:44.283+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:25:44.654+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:25:44.661+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:25:44.662+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:25:45.453+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:25:47.380+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:25:47.382+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:25:47.382+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:25:47.383+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:25:47.435+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:25:47.436+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:25:47.437+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:25:47.438+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:25:48.500+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:25:48.501+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:25:48.501+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:25:48.502+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:25:50.070+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:25:50.071+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:25:50.072+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:25:50.073+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:25:51.184+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:25:51.185+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:25:51.186+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:25:51.186+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:25:52.298+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:25:52.299+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:25:52.299+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:25:52.300+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:25:53.512+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:25:53.513+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:25:53.514+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:25:53.514+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:25:54.726+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:25:54.727+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:25:54.728+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:25:54.729+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:25:55.840+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:25:56.158+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:25:56.159+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:25:56.160+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:25:56.969+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:25:56.970+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:25:56.970+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:25:56.971+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:25:58.082+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:25:58.083+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:25:58.084+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:25:58.084+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:25:59.197+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:25:59.198+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:25:59.199+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:25:59.200+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:26:00.414+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:26:00.415+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:26:00.416+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:26:00.417+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:26:01.394+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 48 for task streaming_to_kafka (Task received SIGTERM signal; 352)
[2023-10-30T11:26:01.427+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=352, status='terminated', exitcode=1, started='11:25:02') (352) terminated with exit code 1
[2023-10-30T11:26:01.429+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 143
